bin/sparkR2.cmd

```diff
@@ -18,7 +18,7 @@ rem limitations under the License.
 rem
```

Comment:
it looks like we should add this to the appveyor list...

Reply:
Do I need to action on this or just wait for it to be approved?

Reply:
We could add `bin/*.cmd` to here: 
https://github.com/apache/spark/blob/828fab03567ecc245a65c4d295a677ce0ba26c19/appveyor.yml#L27-L35